.eslintrc

```diff
@@ -0,0 +1,32 @@
+{
+  "root": true,
+  "parser": "@babel/eslint-parser",
+  "plugins": [],
+  "extends": [
+    "airbnb",
+    "eslint:recommended"
+  ],
```

Comment:
I see different plugins as the one in the .eslintrc file of the Trello ticket, is there any particular reason for it?